torch/csrc/jit/python/script_init.cpp

```diff
@@ -1094,21 +1110,72 @@ void initJitScriptBindings(PyObject* module) {
 
   py::class_<CompilationUnit, std::shared_ptr<CompilationUnit>>(
       m, "CompilationUnit")
-      .def(py::init<>())
+      .def(
+          py::init([](const std::string& lang, const uint32_t _frames_up) {
+            auto cu = std::make_shared<CompilationUnit>();
+            if (lang.size() > 0) {
+              compilationUnit_py_define(*cu, lang, nullptr, _frames_up);
+            }
+            return cu;
+          }),
+          py::arg("lang") = "",
+          py::arg("_frames_up") = 0)
+
       .def(
           "find_function",
```

Comment:
This function can now return `None`, so I think existing callsites should be audited and checks for `None` inserted if needed. The old implementation crashes in `get_function` if the function doesn't exist.

Reply:
I think it'll be OK. (The change is to align with the C++ API)